ensime_shared/client.py

```diff
@@ -249,7 +249,7 @@ def send_at_position(self, what, useSelection, where="range"):
         self.log.debug('send_at_position: in')
         b, e = self.editor.selection_pos() if useSelection else self.editor.word_under_cursor_pos()
         self.log.debug('useSelection: {}, beg: {}, end: {}'.format(useSelection, b, e))
-        self.send_at_point_req(what, self.editor.path(), b[0], b[1], e[0], e[1], where)
+        self.send_at_range_req(what, self.editor.path(), b[0], b[1], e[0], e[1], where)
```

Comment:
Nice, I think that narrowed responsibility of `send_at_point` is an improvement until we get more type-ful requests. Thanks!